Client.py

```diff
@@ -82,11 +81,19 @@ def handleBombPlantedThisRound(self):
         if self.player.bombsHandler.bombPlantedThisRound:
             self.sendBombToServer(self.player.bombsHandler.bombPlantedThisRound)
             self.player.bombsHandler.bombPlantedThisRound = 0
+
+    def sendBoardToServer(self):
+        connection.Send({"action": "boardToServer", "board": self.player.newBoard})
+
```

Comment:
Won't `"action": "board"` work? Does it collide with previous `"board"` that is used in server?